station/tests/test_quality_ratings.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+
+import numpy as np
+
+import quality_ratings
+
+
+class TestQualityRatings(unittest.TestCase):
+    def test_list_names(self):
+        names = quality_ratings.get_rate_names()
+        self.assertTrue("analog" in names)
+        self.assertTrue("digital" in names)
+
+    def test_get_rate_by_name(self):
+        self.assertIsNotNone(quality_ratings.get_rate_by_name("analog"))
+        self.assertIsNotNone(quality_ratings.get_rate_by_name("digital"))
+
+    def test_analog_rating_on_gaussian_noise_small_sigma(self):
+        img = np.random.normal(scale=1, size=(1000, 1000))
+        rate = quality_ratings.get_rate_by_name("analog")
+        rating = rate(img)
+        self.assertAlmostEquals(1.0, rating, 2)
+
+    def test_analog_rating_on_gaussian_noise_big_sigma(self):
+        img = np.random.normal(scale=20, size=(1000, 1000))
+        rate = quality_ratings.get_rate_by_name("analog")
+        rating = rate(img)
+        self.assertAlmostEquals(0.0, rating, 2)
+
+    def test_analog_rating_on_gaussian_noise_medium_sigma(self):
+        img = np.random.normal(scale=12.7, size=(1000, 1000))
+        rate = quality_ratings.get_rate_by_name("analog")
+        rating = rate(img)
+        self.assertAlmostEquals(0.5, rating, 1)
+
+    def test_analog_rating_on_blank(self):
+        img = np.zeros((1000, 1000))
```

Comment:
Some explanation is required for this and the next test. Why for a perfectly uniform image one rating function returns 1 (perfect) and the other returns 0 (complete junk)?

Reply:
First function is test for "analog" imagery rating. Image is black then it doesn't contains any analog noise.
Second function is test for "digital" imagery rating. This rate function calculate ratio between black  and all pixels. Image is black then it is rate as constans no data.

Reply:
Some explanation is required = write a comment in the text :)

Reply:
Comment just added.